tests/simple/simple.go

```diff
@@ -141,11 +145,16 @@ func MatchValue(tag string, expected *exprpb.Value, actual *exprpb.Value) error
 		// - surfaces sign differences for floating-point zero.
 		// Text marshaling isn't documented as deterministic,
 		// but it appears to be so in practice.
-		text := &proto.TextMarshaler{ExpandAny: true}
-		sExpected := text.Text(expected)
-		sActual := text.Text(actual)
-		if sExpected != sActual {
-			return fmt.Errorf("%s: Eval got [%v], want [%v]", tag, sActual, sExpected)
+		sExpected, err := prototext.Marshal(expected)
+		if err != nil {
+			return fmt.Errorf("prototext.Marshal(%v) failed: %v", expected, err)
+		}
+		sActual, err := prototext.Marshal(actual)
+		if err != nil {
+			return fmt.Errorf("prototext.Marshal(%v) failed: %v", actual, err)
+		}
+		if !bytes.Equal(sExpected, sActual) {
+			return fmt.Errorf("%s: Eval got [%v], want [%v]", tag, string(sActual), string(sExpected))
```

Comment:
Consider whether to use the protocmp package here, now that we've switched to the new proto API. At least put in a note to do it in the future.

Reply:
I added a TODO. A per-byte comparison should be pretty accurate since I'm not sure we want to account for list element out of order, etc. at this time.

Reply:
This looks like a per-byte comparison of the utf8 encoded textproto - i.e. a string comparison, as before. We need to be sensitive to list element order - I think you meant map?  In any case, protocomp looks to be exactly what we want here.

Reply:
Oh, the protocmp can ignore list element order as well, that's why I was commenting that I didn't think we needed it yet. I'm happy to change it, but didn't want to do so until after I get the cel-go conformance working with the latest cel-spec.